Preparing for Intern's arrival

markAtMicrosoft opened the pull request:
<!-- Enter a brief description/summary of your PR here. What does it fix/what does it change/how was it tested (even manually, if necessary)? -->
## Summary of the Pull Request

<!-- Other than the issue solved, is this relevant to any other issues/existing PRs? --> 
## References

<!-- Please review the items on the PR checklist before submitting-->
## PR Checklist
* [] Applies to #xxx
* [] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA
* [] Tests added/passed
* [] Requires documentation to be updated
* [] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

<!-- Provide a more detailed description of the PR, other things fixed or any additional comments/features here -->
## Detailed Description of the Pull Request / Additional comments

<!-- Describe how you validated the behavior. Add automated tests wherever possible, but list manual validation steps taken as well -->
## Validation Steps Performed

crutkas commented on doc/Interns/MyFirstCommit.md:
```diff
@@ -0,0 +1,64 @@
+4.	Clone the repo: ([PowerToys on Github](https://github.com/microsoft/PowerToys))
+5.	Change to Branch ***interns/dev***
+	1.	git clone https://github.com/microsoft/PowerToys.git
+5.	Open the "docs/interns/MyFirstCommit.md" document
```
I would do alias.md or alias/MyFirstCommit.md

crutkas commented on doc/Interns/MyFirstCommit.md:
```diff
@@ -0,0 +1,64 @@
+
+- master - Production branch that is typically the release version.  It is not updated until a new release is ready.  It is updated from the **staging** branch
+- staging - internal and limited testing branch.  This branch would be used by sponsors (like Clint) and internal testers to try out the application.  This is updated somewhat frequently, might be once per week to daily as a release nears.  It is updated from the **dev** branch
+- dev - your intern team sandbox and will update multiple times per day by all team members.
```
Intern/dev branch

crutkas commented on doc/Interns/MyFirstCommit.md:
```diff
@@ -0,0 +1,64 @@
+5.	Open the "docs/interns/MyFirstCommit.md" document
+
+## Branches ##
+In the Garage, we tend to follow a specific branching rules.  We ask that you also follow that same rules.  In a independent project we would have the following branches
```
I’d stress they are inside intern/ not our main root

markAtMicrosoft commented:
Whoops, I created my interns/dev branch off of master.  Guess I should merge in stable.

Made your changes and fixed some grammar.  Also fixed some of the paths.  Want the intern work to go into /interns/users/alias/branchname.

Please check it again.  Let me know if this document is in the spirit of what you want.  I added a few things at the front about the garage typical structure before going into what we will do this time.  Hopefully not too long winded.

We'll need a slide on your deck explaining where to go find this document.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________
From: Clint Rutkas <notifications@github.com>
Sent: May 2, 2020 4:19 PM
To: microsoft/PowerToys <PowerToys@noreply.github.com>
Cc: Mark Schramm (Fusionware) <v-schm@microsoft.com>; Assign <assign@noreply.github.com>
Subject: Re: [microsoft/PowerToys] Preparing for Intern's arrival (#2627)


@crutkas approved this pull request.

Small nits.

Our main dev branch is master. Our release branch is stable.

________________________________

In doc/Interns/MyFirstCommit.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerToys%2Fpull%2F2627%23discussion_r419019953&data=02%7C01%7Cv-schm%40microsoft.com%7C7560553b473a4d3ae72908d7eeef4b01%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637240583835582659&sdata=0WlFEwWMCxY0sS0PCjSfSqhJcT0Pe1JQyoc%2BseDszco%3D&reserved=0>:

> @@ -0,0 +1,64 @@

+# PowerToys - My First Commit #

+

+## Getting Setup ##

+

+1.     You will need to install Git Bash.  ([Git tools download here](https://git-scm.com/download/win))

+2.     Create a repo folder on your system.

+3.     Open GitBash in that repo folder.

+4.     Clone the repo: ([PowerToys on Github](https://github.com/microsoft/PowerToys))

+5.     Change to Branch ***interns/dev***

+       1.      git clone https://github.com/microsoft/PowerToys.git

+5.     Open the "docs/interns/MyFirstCommit.md" document



I would do alias.md or alias/MyFirstCommit.md

________________________________

In doc/Interns/MyFirstCommit.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerToys%2Fpull%2F2627%23discussion_r419020109&data=02%7C01%7Cv-schm%40microsoft.com%7C7560553b473a4d3ae72908d7eeef4b01%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637240583835587639&sdata=7pqjOjj6elZfr%2BQM0o6CQCpud9NWyvhCe2URe0LKqmk%3D&reserved=0>:

> +## Getting Setup ##

+

+1.     You will need to install Git Bash.  ([Git tools download here](https://git-scm.com/download/win))

+2.     Create a repo folder on your system.

+3.     Open GitBash in that repo folder.

+4.     Clone the repo: ([PowerToys on Github](https://github.com/microsoft/PowerToys))

+5.     Change to Branch ***interns/dev***

+       1.      git clone https://github.com/microsoft/PowerToys.git

+5.     Open the "docs/interns/MyFirstCommit.md" document

+

+## Branches ##

+In the Garage, we tend to follow a specific branching rules.  We ask that you also follow that same rules.  In a independent project we would have the following branches

+

+- master - Production branch that is typically the release version.  It is not updated until a new release is ready.  It is updated from the **staging** branch

+- staging - internal and limited testing branch.  This branch would be used by sponsors (like Clint) and internal testers to try out the application.  This is updated somewhat frequently, might be once per week to daily as a release nears.  It is updated from the **dev** branch

+- dev - your intern team sandbox and will update multiple times per day by all team members.



Intern/dev branch

________________________________

In doc/Interns/MyFirstCommit.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerToys%2Fpull%2F2627%23discussion_r419020175&data=02%7C01%7Cv-schm%40microsoft.com%7C7560553b473a4d3ae72908d7eeef4b01%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637240583835592616&sdata=WI2Xw8BgchNdZWFVFZphK7NXEFbzxK0ZX13knSJvXhs%3D&reserved=0>:

> @@ -0,0 +1,64 @@

+# PowerToys - My First Commit #

+

+## Getting Setup ##

+

+1.     You will need to install Git Bash.  ([Git tools download here](https://git-scm.com/download/win))

+2.     Create a repo folder on your system.

+3.     Open GitBash in that repo folder.

+4.     Clone the repo: ([PowerToys on Github](https://github.com/microsoft/PowerToys))

+5.     Change to Branch ***interns/dev***

+       1.      git clone https://github.com/microsoft/PowerToys.git

+5.     Open the "docs/interns/MyFirstCommit.md" document

+

+## Branches ##

+In the Garage, we tend to follow a specific branching rules.  We ask that you also follow that same rules.  In a independent project we would have the following branches



I’d stress they are inside intern/ not our main root

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerToys%2Fpull%2F2627%23pullrequestreview-404567585&data=02%7C01%7Cv-schm%40microsoft.com%7C7560553b473a4d3ae72908d7eeef4b01%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637240583835597601&sdata=GWo6D5IG5xJNN9E46Dt1auYy1l08uZz3%2B8lQKMxlrKY%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAIFKUO4SXP4GHAEFHUGJKATRPSTA3ANCNFSM4MX3WFRA&data=02%7C01%7Cv-schm%40microsoft.com%7C7560553b473a4d3ae72908d7eeef4b01%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637240583835597601&sdata=PnHwcO1yitac9Pw4hYm6s0uFa69%2B3T3gM1LySMIyQaM%3D&reserved=0>.

crutkas commented:
Master would be the preferred branch,

markAtMicrosoft commented:
Ok, I think last push to the PR.  If you like it, set it free.

We will need to provide a link to the file once merged that shows that document so they can begin.

Until we put in in the interns/dev branch, this is what the path will sort of look like.

https://github.com/microsoft/PowerToys/blob/interns/dev/doc/Interns/MyFirstCommit.md<https://github.com/microsoft/PowerToys/blob/interns/users/markAtMicrosoft/git101/doc/Interns/MyFirstCommit.md>

I do mention Mark in the there so you might want to point out who that guy is.

Don't forget, we should put in those PR rules for the interns/dev branch so nobody can bypass things (after we merge this current PR because don't have any codeowners until it's completed)



Sent from Outlook<http://aka.ms/weboutlook>

________________________________
From: Clint Rutkas <notifications@github.com>
Sent: May 3, 2020 7:59 AM
To: microsoft/PowerToys <PowerToys@noreply.github.com>
Cc: Mark Schramm (Fusionware) <v-schm@microsoft.com>; Assign <assign@noreply.github.com>
Subject: Re: [microsoft/PowerToys] Preparing for Intern's arrival (#2627)


Master would be the preferred branch,

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerToys%2Fpull%2F2627%23issuecomment-623123005&data=02%7C01%7Cv-schm%40microsoft.com%7C523bb296ee924463e9ea08d7ef729c5d%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637241147836267943&sdata=wWmUBYdY21tAaEL72uD90Z2r0b0qmnXr3OlH9zp0rYI%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAIFKUO2FO4OXFHEMSMN6MDTRPWBF3ANCNFSM4MX3WFRA&data=02%7C01%7Cv-schm%40microsoft.com%7C523bb296ee924463e9ea08d7ef729c5d%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637241147836277901&sdata=SfiW7fRxA87fmqBAhdCPSmTpnyL6017f0Ncnm9Z%2FH7I%3D&reserved=0>.

crutkas commented:
LGTM

markAtMicrosoft commented:
???

Mark Schramm
Technical Advisor
Microsoft Garage Intern Program
________________________________
From: Clint Rutkas <notifications@github.com>
Sent: Sunday, May 3, 2020 9:40:20 AM
To: microsoft/PowerToys <PowerToys@noreply.github.com>
Cc: Mark Schramm (Fusionware) <v-schm@microsoft.com>; Assign <assign@noreply.github.com>
Subject: Re: [microsoft/PowerToys] Preparing for Intern's arrival (#2627)


LGTM

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerToys%2Fpull%2F2627%23issuecomment-623139638&data=02%7C01%7Cv-schm%40microsoft.com%7C720bc93f6708416e26c008d7ef80abaf%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637241208224510487&sdata=UehBKMrQPzh%2FHCX84mWrVhFoBkYEgksv9JMzVhiApKI%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAIFKUO33UHYB5RZZBZLUA4DRPWM7JANCNFSM4MX3WFRA&data=02%7C01%7Cv-schm%40microsoft.com%7C720bc93f6708416e26c008d7ef80abaf%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637241208224520444&sdata=j7xF6F%2BdQdH0kT3y0AL%2BuqvbCQ1xp%2F78EY80wQ1ArLk%3D&reserved=0>.
